frontend/libs/shared/util-forms/src/lib/services/qro-dialog.service.ts

```diff
@@ -0,0 +1,98 @@
+import { Injectable, TemplateRef } from '@angular/core';
+import { MatDialog, MatDialogConfig, MatDialogRef } from '@angular/material/dialog';
+import { ConfirmationDialogComponent } from '@qro/shared/ui-confirmation-dialog';
+import { ComponentType } from '@angular/cdk/portal';
+import { Observable } from 'rxjs';
+import { filter, switchMap } from 'rxjs/operators';
+import { ContactPersonDialogComponent } from '@qro/client/ui-contact-person-detail';
+
+export interface ContactPerson {
+  id?: string;
+  lastName?: string;
+  firstName?: string;
+  phone?: string;
+  email?: string;
+}
+
+export interface ContactPersonDialogData {
+  contactPerson?: ContactPerson;
+}
+
+export interface ConfirmDialogData {
+  text?: string;
+  title?: string;
+  abortButtonText?: string;
+  confirmButtonText?: string;
+}
+
+@Injectable({
+  providedIn: 'root',
+})
+export class QroDialogService {
```

Comment:
Ich finde es ein bisschen unglücklich, dass dieser Service QroDialogService heißt, was einem allgemeinen Service für Dialoge klingt, er aber eigentlich vor allem für den speziellen Kontaktperson-Dialog gedacht ist (siehe auch die interfaces darin). Vielleicht wäre es gut, ihn entweder umzubenennen oder die Kontakt-spezifischen Dinge in einen separaten Service auszulagern

Reply:
Meine Idee war es schon das man hier auch zukuenftig Dialoge sammelt. Also eher ein allgemeiner Service. Wir koennen das aber auch spezifischer gestalten.

Reply:
ist jetzt contact-dialog in der client lib